service/dependencies.gradle

```diff
@@ -36,14 +36,14 @@ dependencies {
   // Get stairway via TCL
   implementation("bio.terra:terra-common-lib:1.1.6-SNAPSHOT")
   // sam is not semantically versioned so force the version here
-  implementation force: true, group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.13", version: "0.1-73bc2d1"
+  implementation force: true, group: "org.broadinstitute.dsde.workbench", name: "sam-client_2.13", version: "v0.0.224-c536ffc-SNAP"
```

Comment:
TODO: change to non-SNAP once https://github.com/broadinstitute/sam/pull/1430 is merged